pycls/core/io.py

```diff
@@ -13,9 +13,12 @@
 import sys
 from urllib import request as urlrequest
 
-from iopath.common.file_io import g_pathmgr
+from iopath.common.file_io import PathManagerFactory
 
 
+# instantiate global path manager for pycls
+pathmgr = PathManagerFactory.get()
```

Comment:
Good to make a local version for the library. More preferable to have only one instance per run but I'm not going to insist on that right now